src/utils/vp.ts

```diff
@@ -54,7 +57,10 @@ export async function getVp(
       if (addresses.length === 0) return {};
     }
 
-    addresses = addresses.map(getAddress);
+    addresses = getFormattedAddressesByProtocol(
+      addresses,
+      strategy.supportedProtocols ?? DEFAULT_SUPPORTED_PROTOCOLS
+    );
```

Comment:
Sorry for not noticing earlier 🙈 With previous changes, we were not  modifying the length of `addresses`. If an address is not valid, it throws an error; however, now with getFormattedAddressesByProtocol it filters addresses. Can this cause any problems? 


I am mostly worried if we don't send back a response to an address, it will asume `0` as voting power

Reply:
it maybe very rare case to see invalid address, but better to throw instead of filtering 

Would it cause any problem if we throw error  instead of filtering? we could write a reusuable function for `validateAddressType`

Reply:
now returning an error on invalid address/address from unsupported protocols